thirdparty_containers/jenkins/playlist.xml

```diff
@@ -15,7 +15,9 @@
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
 	<test>
 		<testCaseName>jenkins_test</testCaseName>
-	<command>docker run --rm -v $(JDK_HOME):/java  adoptopenjdk-jenkins-test:latest "jvm pos neg"; \
+	<command>docker run --rm -v $(JDK_HOME):/java --name jenkins-test adoptopenjdk-jenkins-test:latest "jvm pos neg"; \
+			 docker cp jenkins-test:/jenkins/cli/target/surefire-reports $(REPORTDIR)/external_test_reports; \
+			 docker cp jenkins-test:/jenkins/core/target/surefire-reports $(REPORTDIR)/external_test_reports; \		
```

Comment:
should we remove the image at the end?

Reply:
Given that we have space issues on test machines, I think it might be the right thing to do (remove image, at least in the case of successful runs).

Reply:
Good catch @smlambert ! I missed the step to remove the image! Added it.